dist/index.html

```diff
@@ -99,8 +99,85 @@
         </div>
       </section>
 
-      <section class="slider">
-        Navbar And Slider
+      <section
+        class="slider pt-0 bg-cover bg-center h-screen relative bg-c800 relative"
+        style="
+          background-image: url('assets/img/header-slider/slider.jpg');
+          height: 80vh;
+        "
+      >
+        <div
+          class="overlay absolute h-full w-full"
+          style="background-color: rgb(37, 65, 81, 0.8);"
+        >
+          <div class="container h-full flex flex-col">
+            <nav class="flex">
```

Comment:
Great work for using` flex-box` to center items vertically and horizontally. 👍 💯